[Inductor] Support Shape Padding for aten.mm in Inductor

jiawenliu64 opened the pull request:
Summary: Support shape padding for aten.mm in Inductor (originally from [#88709](https://github.com/pytorch/pytorch/pull/88709))

Differential Revision: D41315078



cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

jianyuh reviewed (approved):
LGTM (originally from https://github.com/pytorch/pytorch/pull/88709 )!

jiawenliu64 commented:
@pytorchbot merge

Commit:
[Inductor] Support Shape Padding for aten.mm in Inductor (#89086)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/89086

Support shape padding for aten.mm in Inductor

Reviewed By: jianyuh

Differential Revision: D41315078

fbshipit-source-id: 561ce3a81e286fb711580e057f60dcff9919bdf4

jiawenliu64 commented:
@pytorchbot merge
